Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.2.2'
+ruby '3.1.3'
```

Comment:
- [ ] Great work implementing the features for reservations. 💯 However, please kindly ensure to update the Ruby version back to `3.2.2`. 🤓

Reply:
thanks alot already did that !